langchain4j-azure-aisearch-spring-boot-starter/src/main/java/dev/langchain4j/azure/aisearch/spring/Properties.java

```diff
@@ -0,0 +1,22 @@
+package dev.langchain4j.azure.aisearch.spring;
+
+import dev.langchain4j.rag.content.retriever.azure.search.AzureAiSearchQueryType;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@Getter
+@Setter
+@ConfigurationProperties(prefix = Properties.PREFIX)
+public class Properties {
+
+    static final String PREFIX = "langchain4j.azure.ai-search";
+
+    String endpoint;
+    String apiKey;
+    int dimensions;
+    boolean createOrUpdateIndex;
+    int maxResults = 1;
```

Comment:
```suggestion
    int maxResults = 3;
```
We use "3" as a default value everywhere. Is there a reason why we should use 1 here?